deployment/README.md

```diff
@@ -1,72 +1,129 @@
-# Ocelot.Social Deploy And Rebranding
+# Ocelot.Social Deployment And Rebranding
 
-[![Build Status Publish](https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/actions/workflows/publish.yml/badge.svg)](https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/actions)
-[![MIT License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding/blob/LICENSE.md)
-[![Discord Channel](https://img.shields.io/discord/489522408076738561.svg)](https://discord.gg/AJSX9DCSUA)
-[![Open Source Helpers](https://www.codetriage.com/ocelot-social-community/ocelot-social-deploy-rebranding/badges/users.svg)](https://www.codetriage.com/ocelot-social-community/ocelot-social-deploy-rebranding)
-
-This repository is an in use template to rebrand, configure, and deploy [ocelot.social](https://github.com/Ocelot-Social-Community/Ocelot-Social) networks.
-The forked original repository is [Ocelot-Social-Deploy-Rebranding](https://github.com/Ocelot-Social-Community/Ocelot-Social-Deploy-Rebranding).
-
-<!-- markdownlint-disable MD033 -->
-<p align="center">
-  <a href="https://ocelot.social" target="_blank"><img src="branding/static/img/custom/logo-squared.svg" alt="Ocelot-Social" width="40%" height="40%"></a>
-</p>
-<!-- markdownlint-enable MD033 -->
+The `deployment` directory is about how to change the look and feel of ocelot.social (called branding) and how to get the software running on a kubernetes cluster (deployment).
 
 ## Live demo
 
 __Try out our deployed [development environment](https://stage.ocelot.social).__
 
-Visit our staging networks:
-
-- central staging network: [stage.ocelot.social](https://stage.ocelot.social)
-<!-- - rebranded staging network: [rebrand.ocelot.social](https://stage.ocelot.social). -->
+This environment is the vanilla ocelot without branding and with seed data.
```

Comment:
Because this is a readme in the general deployment, where `yunite.me` and other will be placed, `vanilla ocelot` doesn't fit.